src/service/url-expander/expander.js

```diff
@@ -125,7 +125,7 @@ export class Expander {
     let ignoringChars = false;
     let nextArgShouldBeRaw = false;
 
-    const evaluateNextLevel = () => {
+    const evaluateNextLevel = depth => {
```

Comment:
instead of passing a depth, we only need a boolean

Reply:
Done.